libs/networking/http_helpers.mli

```diff
@@ -12,6 +12,7 @@ val get_async :
 val post_async :
   body:string ->
   ?headers:(string * string) list ->
+  ?chunked:bool ->
```

Comment:
maybe a small comment about chunked is?
(many of the PA programmers, including me, are not super familiar with the http world,
so I can understand the body and headers of a rerquest, but chunked?